src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -2398,53 +2399,115 @@ def delete_ssl_cert(cmd, resource_group_name, certificate_thumbprint):
     raise CLIError("Certificate for thumbprint '{}' not found".format(certificate_thumbprint))
 
 
-def import_ssl_cert(cmd, resource_group_name, name, key_vault, key_vault_certificate_name):
-    Certificate = cmd.get_models('Certificate')
+def import_ssl_cert(cmd, resource_group_name, name, certificate=None, key_vault_certificate_name=None,
+                    key_vault=None):
+    Certificate_Model = cmd.get_models('Certificate')
     client = web_client_factory(cmd.cli_ctx)
     webapp = client.web_apps.get(resource_group_name, name)
     if not webapp:
```

Comment:
We should have a validator in validator.py now that should handle this? Can you check? & if so remove this to use the validator?

Reply:
BTW if you want keep the disruptive changes to a minimum - its ok to leave it here for now, but make sure we don't  follow this pattern on the new code/ commands.